lib/pages/settings_dashboard/settings/settings_view_style.dart

```diff
@@ -1,17 +1,21 @@
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
 import 'package:flutter/material.dart';
 
 class SettingsViewStyle {
+  static ResponsiveUtils responsiveUtils = getIt.get<ResponsiveUtils>();
   static const double iconSize = 24.0;
 
   static const double fontSizeAvatar = 9 * 2.5;
 
   static const AlignmentGeometry alignment = AlignmentDirectional.centerStart;
 
-  static const EdgeInsetsDirectional titlePadding =
-      EdgeInsetsDirectional.only(start: 16);
-
+  static const EdgeInsetsDirectional titlePaddingMobile =
+      EdgeInsetsDirectional.only(top: 12, bottom: 11);
+  static const EdgeInsetsDirectional titlePaddingWeb =
+      EdgeInsetsDirectional.symmetric(horizontal: 16, vertical: 8);
   static EdgeInsetsDirectional itemBuilderPadding =
-      const EdgeInsetsDirectional.all(16.0);
+      const EdgeInsetsDirectional.only(top: 16.0, bottom: 16.0, start: 8.0);
```

Comment:
should 8 for all

Reply:
It has 16 at the top and bottom 
![image](https://github.com/user-attachments/assets/8fef5078-1a5b-44c4-862c-b44668110b56)